sentry-android-core/src/main/java/io/sentry/android/core/AndroidSerializer.java

```diff
@@ -42,6 +43,7 @@ private Gson provideGson() {
             Device.DeviceOrientation.class, new OrientationDeserializerAdapter(logger))
         .registerTypeAdapter(SentryLevel.class, new SentryLevelSerializerAdapter(logger))
         .registerTypeAdapter(SentryLevel.class, new SentryLevelDeserializerAdapter(logger))
+        .registerTypeAdapter(Contexts.class, new ContextsDeserializerAdapter(logger))
```

Comment:
We could have a single public class in the `io.sentry.core.adapter` package that register all of these adapters and make them all internal. What do you think?

Reply:
sounds good